cogs/owner.py

```diff
@@ -19,6 +19,71 @@
 class Owner(commands.Cog, name="owner"):
     def __init__(self, bot):
         self.bot = bot
+    
+    @commands.hybrid_command(
+        name="load",
+        description="Load a cog",
+    )
+    @checks.is_owner()
+    async def load(self, ctx, cog: str):
+         """
+        The bot will load the given cog.
+
+        :param context: The hybrid command context.
+        :param cog: The cog to be loaded.
+        """
+        try:
+            await self.bot.load_extension(f"cogs.{cog}")
+        except Exception as e:
+            embed=discord.Embed(title="load", description=f"Could not load the `{cog}` cog.")
+            await ctx.send(embed=embed)
+            return
+        embed=discord.Embed(title="Load", description=f"Successfully loaded the `{cog}` cog.")
+        await ctx.send(embed=embed)
+
+
+    @commands.hybrid_command(
+        name="unload",
+        description="Unloads a cog.",
+    )
+    @checks.is_owner()
+    async def unload(self, ctx, cog: str):
+        """
+        The bot will unload the given cog.
+
+        :param context: The hybrid command context.
+        :param cog: The cog to be unloaded.
+        """
+        try:
+            await self.bot.unload_extension(f"cogs.{cog}")
+        except Exception as e:
+            embed=discord.Embed(title="Unload", description=f"Could not unload the `{cog}` cog.")
+            await ctx.send(embed=embed)
+            return
+        embed=discord.Embed(title="Unload", description=f"Successfully loaded the `{cog}` cog.")
+        await ctx.send(embed=embed)
+
+    @commands.hybrid_command(
+        name="reload",
+        description="Reloads a cog.",
+    )
+    @checks.is_owner()
+    async def reload(self, ctx, cog: str):
+        """
+        The bot will reload the given cog.
+
+        :param context: The hybrid command context.
+        :param cog: The cog to be reloaded.
+        """
+        try:
+            await self.bot.unload_extension(f"cogs.{cog}")
+            await self.bot.load_extension(f"cogs.{cog}")
```

Comment:
Any specific reasons on using `unload_extension` and `load_extension` instead of just [`reload_extension`](https://discordpy.readthedocs.io/en/stable/ext/commands/api.html#discord.ext.commands.Bot.reload_extension)?

Reply:
This was pretty old code lol, I'll make changes